### PR TITLE
Register btc transaction logs

### PR DESCRIPTION
**Pull request by marcos-iov**
- Cleanup registerBtcTransaction method
- Add logs to registerBtcTransaction. Log btc tx hash without witness

*fed:FINGERROOT-5.0.0-rc*
*fit:FINGERROOT-5.0.0-rc*
*ci:FINGERROOT-5.0.0-rc*

### CODE REVIEW

**Review by jeremy-then — DISMISSED** [review]
LGTM.

**Comment by marcos-iov** [comment]
pipeline:run

**Review by josedahlquist — DISMISSED** [review]

File: rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java
[the remark this answers is not in the document]
Comment (marcos-iov) [reply rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java:1]: Done

**Review by josedahlquist — APPROVED** [review]
Lgtm

**Pull request merged**